frontend-project/src/pages/tags/new/index.tsx

```diff
@@ -49,13 +59,14 @@ const TagNewForm: FC<TagNewFormProps> = ({ dispatch }) => {
                 name="name"
                 rules={[
                   {
-                    // required: true,
+                    required: true,
                     message: formatMessage({ id: 'tags-new.form.name.required-error' }),
                   },
                 ]}
               >
                 <Input placeholder={formatMessage({ id: 'tags-new.form.name.placeholder' })} />
               </Form.Item>
+              {props.createdTag.tag?.name && <TagMessage content={props.createdTag.tag?.name} />}
```

Comment:
Zupełnie nie rozumiem po co to wprowadziłeś. Przecież `form.setFields(Object.entries(response).map(([name, errors]) => ({ name, errors }))` już zapewniał wyświetlanie komunikatu błędu, a sposób wyświetlania komunikatu błędu był skuteczny. Jak będzie to wyglądać, gdy będziesz miał 10 pól w formularzu?

Reply:
To musiał bym takie sprawdzenie dodać 10 razy w formularzu z odpowiednim property po props.createdTag.tag? 
<img width="1434" alt="Zrzut ekranu 2020-10-27 o 07 17 33" src="https://user-images.githubusercontent.com/9075439/97264083-8984c080-1824-11eb-99c9-1729162da39d.png">
Poprawiłem, i przy okazji pozmieniałem nazwę tej właściwości createdTag ponieważ myślę, że jest nieodpowiednia szczególnie w przypadku niepowodzenia w tworzeniu nowego tag, jest to tagState. U mnie jeszcze nie do końca to działa, ponieważ żeby error się wyświetlił należy kliknąć 2 razy w button, zajrzę do tego popołudniu chyba, że ktoś rozwiąże ten problem wcześniej :)

Reply:
Po ostatnich zmianach (wyciągnięcia mapowania poza submit oraz utworzenia default state w TagModels) u mnie działa prawidłowo. Odpowiedni Error wyświetla się już przy pierwszym kliknięciu Zapisz.